CAD-4437 Include UTxO HD installation/usage instructions in READMEs (#4079)
input-output-hk/cardano-node: README.rst

* ``--validate-db`` - Flag to revalidate all on-disk database files

> * ``--in-memory-ledger-db-backend`` - Optionally use the in-memory backend of the UTxO HD feature: store the complete ledger state in memory. Incompatible with ``--lmdb-ledger-db-backend``.

Reviewer: ```suggestion
* ``--in-memory-ledger-db-backend`` - Use the in-memory backend to store the ledger state. Incompatible with ``--lmdb-ledger-db-backend``.
```

Reviewer: Should we wrap this up to 80 characters?

> * ``--lmdb-ledger-db-backend`` - Optionally use the LMDB backend of the UTxO HD feature: store parts of the ledger state on disk. Incompatible with ``--in-memory-ledger-db-backend``. The node uses the LMDB backend by default if no ``--*-db-backend`` flags are set.

Reviewer: ```suggestion
* ``--lmdb-ledger-db-backend`` - Use the LMDB backend to store parts of the ledger state on disk. Incompatible with ``--in-memory-ledger-db-backend``. The node uses the LMDB backend by default if no ``--*-db-backend`` flags are set.
```

>   * ``--lmdb-mapsize`` - Optionally set the mapsize (maximum database size) for the LMDB backend. By default, the mapsize (maximum database size) of the backend is set to 16 Gigabytes. Warning: if the database size exceeds the given mapsize, the node will abort. Therefore, the mapsize should be set to a value high enough to guarantee that the maximum database size will not be reached during the expected node uptime.

Reviewer: ```suggestion
  * ``--lmdb-mapsize`` - Set the maximum database size (mapsize) for the LMDB backend. By default, the mapsize of the backend is set to 16 Gigabytes. Warning: if the database size exceeds the given mapsize, the node will abort. Therefore, the mapsize should be set to a value high enough to guarantee that the maximum database size will not be reached during the expected node uptime.
```
